docker-compose.yml

```diff
@@ -17,6 +17,16 @@ services:
       HUMHUB_DEBUG: 1
       HUMHUB_PROTO: 'https'
       HUMHUB_HOST: 'localhost'
+      # Mailer install setup
+      HUMHUB_MAILER_SYSTEM_EMAIL_ADDRESS: 'noreply@example.com'
+      HUMHUB_MAILER_SYSTEM_EMAIL_NAME: 'HumHub'
+      HUMHUB_MAILER_TRANSPORT_TYPE: 'smtp'
+      HUMHUB_MAILER_HOSTNAME: 'mailer'
+      HUMHUB_MAILER_PORT: 1025
+      #HUMHUB_MAILER_USERNAME:
+      #HUMHUB_MAILER_PASSWORD:
+      #HUMHUB_MAILER_ENCRYPTION:
+      HUMHUB_MAILER_ALLOW_SELF_SIGNED_CERTS: 0
```

Comment:
shouldn't that be enabled by default? Allowing self-signed certs per default is against good practices. Also, mailcatches is not using TLS in this configuration, isn't it? So this setting isn't doing anything right?

Reply:
It's against good practices **in production** but mailcatcher is definitely not something that should be using in production.
It may be useful though for testing in development that an app can manage secure connections.

TLS is disabled by default, so you are correct this setting does nothing. It's just here for documentation and can be removed if you prefer.

Reply:
"It's against good practices in production but mailcatcher is definitely not something that should be using in production." I totally agree. Just to clarify: I was just wondering about the fact that `ALLOW_SELF_SIGNED_CERTS = false` is set explicitly. This implies to me that the default is `ALLOW_SELF_SIGNED_CERTS = true` which is contra-intuitive to me. I haven't looked at the actual default and because it is set to 0 anyways it is fine for me.